lib/batch.js

```diff
@@ -0,0 +1,3 @@
+import {unstable_batchedUpdates} from 'react-dom';
```

Comment:
Please rename file to `unstable_batchedUpdates.js` so that the name of the file matches the name of what is exported.